src/Feature/Layout/Feature_Layout.re

```diff
@@ -134,143 +139,227 @@ let update = (~focus, model, msg) => {
 
   | Command(MoveLeft) =>
     switch (focus) {
-    | Some(focus) => (model, Focus(moveLeft(focus, activeTree(model))))
+    | Some(Center(focus)) =>
+      let newFocus = model |> activeTree |> moveLeft(focus);
+      if (newFocus == focus) {
+        (model, Focus(Left));
+      } else {
+        (model, Focus(Center(focus)));
```

Comment:
I'm able to move right to left with this change - should this be `Focus(Center(newFocus))`?

Reply:
Indeed it should. Sorry, I don't know how I missed that. Fixed in https://github.com/onivim/oni2/pull/1924/commits/f9ddab0125d272f8b27acdf34022a2ae3e8704db